mmv1/products/compute/terraform.yaml

```diff
@@ -2227,6 +2227,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         custom_flatten: 'templates/terraform/custom_flatten/health_check_log_config.go.erb'
   RegionUrlMap: !ruby/object:Overrides::Terraform::ResourceOverride
     examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "url_map_sample"
+        primary_resource_type: "google_compute_region_url_map"
+        primary_resource_id: "url_map"
+        vars:
+          health_check: "health-check"
+          backend_service: "backend-service"
+          lb_target_proxy: "lb-target-proxy"
+          url_map: "url-map"
+        skip_docs: true
+        min_version: beta
```

Comment:
If this is beta-only you need to use `provider = google-beta` in the config

Reply:
This is not beta-only. Included skip_test: true in terraform.yaml

Reply:
Then we need to remove min_version: beta, that prevents it from being marked as a GA test